### PR TITLE
splash screen

### LINKED ISSUE #6: Splash Screen

### PR DESCRIPTION
**Pull request by VariantAbdulaziz**

### CODE REVIEW

File: qelem/lib/screens/splash_screen.dart [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+
+class Splash extends StatefulWidget {
```
Comment (brukted) [inline qelem/lib/screens/splash_screen.dart]: Could you name it "SplashScreen" instead of splash?

Comment (VariantAbdulaziz) [reply qelem/lib/screens/splash_screen.dart]: will do

File: qelem/lib/screens/splash_screen.dart [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+  void _navigateToHome() async {
+    await Future.delayed(const Duration(seconds: 2), () {});
+    Navigator.pushReplacement(
+        context, MaterialPageRoute(builder: (context) => widget.nextScreen));
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final productNameStyle = Theme.of(context).textTheme.titleLarge;
```
Comment (brukted) [inline qelem/lib/screens/splash_screen.dart]: its Headline 1 from primary text theme not titleLarge.
```suggestion
    final productNameStyle = Theme.of(context).primaryTextTheme.headline1;
```

File: qelem/lib/screens/splash_screen.dart [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    final ownerStyle = Theme.of(context).textTheme.subtitle2;
```
Comment (brukted) [inline qelem/lib/screens/splash_screen.dart]:
```suggestion
    final ownerStyle = Theme.of(context).primaryTextTheme.subtitle2;
```

**Review by brukted — CHANGES_REQUESTED** [review]

File: qelem/lib/screens/splash_screen.dart [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  void _navigateToHome() async {
+    await Future.delayed(const Duration(seconds: 2), () {});
+    Navigator.pushReplacement(
+        context, MaterialPageRoute(builder: (context) => widget.nextScreen));
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final productNameStyle = Theme.of(context).primaryTextTheme.headline1;
+    final ownerStyle = Theme.of(context).primaryTextTheme.subtitle2;
```
Comment (brukted) [inline qelem/lib/screens/splash_screen.dart]:
```suggestion
    final ownerStyle = Theme.of(context).primaryTextTheme.subtitle1;
```

**Review by brukted — APPROVED** [review]

**Review by Biruk-Tassew — APPROVED** [review]

**Review by nawrazi — APPROVED** [review]

**Pull request merged**